crates/oxc_minifier/tests/peephole/inline_single_use_variable.rs

```diff
@@ -226,20 +226,20 @@ fn test_inline_single_use_variable() {
     // cannot be compressed to `function wrapper() { var foo; globalThis[console.log(foo)] = (() => { foo = 1 })() }`
     test(
         "function wrapper() { var foo; var bar = (() => { foo = 1 })(); globalThis[console.log(foo)] = bar }",
-        "function wrapper() { var foo, bar = (() => { foo = 1 })(); globalThis[console.log(foo)] = bar }",
+        "function wrapper() { var foo, bar = (foo = 1, void 0); globalThis[console.log(foo)] = bar }",
     );
     // cannot be compressed to `function wrapper() { let foo; return foo.bar = (() => { foo = {} })(), foo }`
     test(
         "function wrapper() { let foo; const bar = (() => { foo = {} })(); foo.bar = bar; return foo }",
-        "function wrapper() { let foo, bar = (() => { foo = {} })(); return foo.bar = bar, foo }",
+        "function wrapper() { let foo, bar = (foo = {}, void 0); return foo.bar = void 0, foo }",
     );
     test(
         "function wrapper() { let foo = {}; const bar = (() => { console.log() })(); foo.bar = bar; return foo }",
-        "function wrapper() { let foo = {}; return foo.bar = (() => { console.log() })(), foo }",
+        "function wrapper() { let foo = {}, bar = (console.log(), void 0); return foo.bar = void 0, foo; }",
```

Comment:
this result is surprisingly longer, but i think this is correct, `bar` should be removed here